layers.go

```diff
@@ -1399,6 +1402,27 @@ func (r *layerStore) create(id string, parentLayer *Layer, names []string, mount
 		selinux.ReserveLabel(mountLabel)
 	}
 
+	pickLocation := func(volatile, imageStore bool, parentLayer *Layer) layerLocations {
```

Comment:
Can this be made a top-level method? That would, at least in principle, allow writing unit tests :)